src/plugins/intel_cpu/src/transformations/defs.hpp

```diff
@@ -51,5 +51,24 @@ namespace intel_cpu {
 
 #endif
 
+#if defined(OPENVINO_ARCH_ARM64)
+
+#define OV_CPU_REGISTER_PASS_ARM64(MANAGER, PASS) \
```

Comment:
I am already added `CPU_REGISTER_PASS_ARM` above. Do you expect we need to do conditional transformation pipeline for 64bit arch only or the same applicable for 32 bits as well?

Reply:
Deleted